crates/api/api_crud/src/user/my_user.rs

```diff
@@ -30,15 +30,17 @@ pub async fn get_my_user(
   let (
     follows,
     community_blocks,
-    instance_blocks,
+    instance_communities_blocks,
+    instance_persons_blocks,
     person_blocks,
     moderates,
     keyword_blocks,
     discussion_languages,
   ) = lemmy_db_schema::try_join_with_pool!(pool => (
     |pool| CommunityFollowerView::for_person(pool, person_id),
     |pool| CommunityActions::read_blocks_for_person(pool, person_id),
-    |pool| InstanceActions::read_blocks_for_person(pool, person_id),
+    |pool| InstanceActions::read_communities_block_for_person(pool, person_id),
+    |pool| InstanceActions::read_persons_block_for_person(pool, person_id),
     |pool| PersonActions::read_blocks_for_person(pool, person_id),
     |pool| CommunityModeratorView::for_person(pool, person_id, Some(&local_user_view.local_user)),
     |pool| LocalUserKeywordBlock::read(pool, local_user_id),
```

Comment:
Might be worth caching this as it runs a lot of queries, depending how often it gets called for a single user.